### ISSUE TITLE
[Accessibility] Missing Focus Outline in Dark Mode for DropDownButton/Ellipse Button in PropertyGrid

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#13765]
### .NET version

 .NET 10 SDK build: 10.0.100-preview.7.25380.105

### Did it work in .NET Framework?

Not tested/verified

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

Yes, occurs after the PR change: https://github.com/dotnet/winforms/pull/13747

### Issue description

In WinForms, when the application is running in **Dark Mode**, the focus indicator (e.g., dotted line or outline) is not visible when navigating to the `DropDownButton` or` Ellipse Button` controls by keyboard within the `PropertyGrid`. 
This issue does **not occur** in **Classic Mode**, where the focus outline is clearly visible when using the Tab key.

Dark Mode:

https://github.com/user-attachments/assets/27a966f7-14db-4c9a-ad9a-8a16be4846d7

Classic Mode:

<img width="658" height="52" alt="Image" src="https://github.com/user-attachments/assets/726ebacf-cca7-4e5e-8003-0c8ed318cd89" />

<img width="660" height="52" alt="Image" src="https://github.com/user-attachments/assets/a4363d8f-447d-4667-9ac3-d19d06d44327" />

### Steps to reproduce

1. Create Winforms .NET application with targeting .NET 10
2. Add a  PropertyGrid control in form designer
3. Set the SelectedObject property to Form in propertyGrid1's properties
4. Set app under Dark mode in program.cs file:
```
#pragma warning disable WFO5001
      Application.SetColorMode(SystemColorMode.Dark);
#pragma warning restore WFO5001
```
5. Build and run project
6. Tab to any property with DropDownButton/Ellipse Button in PropertyGrid to observe

**Comment by Zheng-Li01** [comment]
Verified the issue with .NET SDK 10.0.100-rc.1.25414.103 build, the issue has been fixed that hove focus outline in DarkMode for DropDownButton/Elipse button in PropertyGrid as below screenshot.
<img width="842" height="317" alt="Image" src="https://github.com/user-attachments/assets/692f14ca-418d-4ab8-9d77-08ec0424aa6b" />

**Comment by Olina-Zhang** [comment]
Verified it in .NET 10.0 RC1 SDK test pass build: 10.0.0-rc.1.25416.112, fixed as following:

<img width="466" height="232" alt="Image" src="https://github.com/user-attachments/assets/f3271479-7f1e-47ec-8ca2-f0b5afcc92fb" />

<img width="470" height="230" alt="Image" src="https://github.com/user-attachments/assets/4cbab540-5cc6-44a6-9093-a0a04dcc27e7" />
